Tests/MeiliSearchUnitTests/DocumentsTests.swift

```diff
@@ -148,13 +148,13 @@ class DocumentsTests: XCTestCase {
 
   func testUpdateDocuments() {
     let jsonString = """
-      {"uid": 0, "indexUid": "movies_test", "status": "enqueued", "type": "documentAddition", "enqueuedAt": "xxx" }
+      {"taskUid":0,"indexUid":"books_test","status":"enqueued","type":"documentAdditionOrUpdate","enqueuedAt":"2022-07-21T21:47:50.565717794Z"}
```

Comment:
Im not sure about removing the spaces between the fields. Do you think it is better?

Reply:
I just copy pasted a real response from the engine, I don't think there is much value about keeping or not those spaces.